packages/styles/table.css

```diff
@@ -103,12 +104,21 @@
 
 .Table--border,
 .Table--border .TableHeader,
+.Table--border .TableFooter,
 .Table--border .TableCell {
   border: 1px solid var(--gray-40);
 }
 
 .cauldron--theme-dark .Table--border,
 .cauldron--theme-dark .Table--border .TableHeader,
+.cauldron--theme-dark .Table--border .TableFooter,
 .cauldron--theme-dark .Table--border .TableCell {
   border-color: var(--stroke-dark);
 }
+
+.TableFooter .TableCell {
+  background: var(--table-header-background-color);
+  font-weight: var(--font-weight-medium);
+  color: var(--table-header-text-color);
+  border-bottom: 2px solid var(--table-row-border-color);
```

Comment:
Is this really supposed to be a 2px border?

Reply:
Good catch it doesn't need 2px bottom border. it already has a defined 1px bottom border